### ISSUE TITLE
passenger-config install-standalone-runtime: cannot find pcre on FreeBSD

### ISSUE THREAD

**Issue by steakknife** [phusion/passenger#1548]
**TL;DR:** Hacky fix
```
env EXTRA_CFLAGS='-I /usr/local/include' EXTRA_LDFLAGS='-L /usr/local/lib' passenger-config install-standalone-runtime
```

FreeBSD port `devel/pcre` -> `pcre-8.37_1                    Perl Compatible Regular Expressions library`
Headers are at `/usr/local/include/pcre*.h`
Libs are at `/usr/local/lib/libpcre.*`

### Output - failure

```
 * Checking for PCRE development headers...
      Found: no

Some required software is not installed.
But don't worry, this installer will tell you how to install them.
Press Enter to continue, or Ctrl-C to abort.

--------------------------------------------

Installation instructions for required software

 * To install PCRE development headers:
   Please download it from http://www.pcre.org/

If the aforementioned instructions didn't solve your problem, then please take
a look at the Users Guide:

  /usr/local/ruby/ruby-2.2.2/lib/ruby/gems/2.2.0/gems/passenger-5.0.10/doc/Users guide Standalone.html
  https://www.phusionpassenger.com/documentation/Users%20guide%20Standalone.html
```

### Output - hacky fix
```
...
Downloading Nginx 1.8.0 source code...
######################################################################## 100.0%
Extracting tarball...
Compiling support libraries (step 1 of 2)...
[**************************************************] -
Compiling Nginx engine (step 2 of 2)...
[**************************************************] -
Compilation finished!
``

**Comment by FooBarWidget** [comment]
I don't think this is a bug. We only look in the standard compiler paths by default. If a library is installed outside the standard compiler paths, you are supposed to modify EXTRA_C(XX)FLAGS and EXTRA_(CXX_)LDFLAGS.
